SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1242,6 +1247,9 @@ AActor* USpatialReceiver::CreateActor(UnrealMetadata* UnrealMetadataComp, SpawnD
 	NewActor->Role = ROLE_SimulatedProxy;
 	NewActor->RemoteRole = ROLE_Authority;
 
+	// Any Actor created here will have been received over the wire as an entity so we can mark it ready.
+	NewActor->SetActorReady(true);
```

Comment:
State should be applied before authority, so I feel this should be set inside ReceiveActor after component data has been applied.

Reply:
Sure, sgtm, this should be harmless enough since it's only setting the property and not triggering any events, but moving it to directly before something like `DispatchBeginPlay` makes more sense generally 👍